app/queries/dropout_student_mutator.rb

```diff
@@ -0,0 +1,23 @@
+class DropoutStudentMutator < ApplicationQuery
+  include AuthorizeSchoolAdmin
+
+  property :id, validates: { presence: true }
+
+  validate :student_must_exist
```

Comment:
I think there should be a validation to prevent dupe requests to mark a student as dropped out. i.e., error out if the student is already dropped out. The service doesn't perform a check, so there probably should be one here.